### ISSUE TITLE
Validate misses flagging a warning about the completeness and accuracy of collection inventories

### ISSUE THREAD

**Issue by mace-space** [NASA-PDS-Incubator/validate#654]
### Checked for duplicates

Yes - I've already checked

### 🐛 Describe the bug

Referential integrity check appears to miss warnings about collection_data.csv and data products in a bundle of raw RSS Voyager data (link to data below).

Transfer manifest contains the following entries:
```
urn:nasa:pds:voyager2_rss_jupiter_raw:browse:vg2j_14mr_1979191t131727::1.0      ./browse/vg2j_14mr_1979191t131727.xml
urn:nasa:pds:voyager2_rss_jupiter_raw:data:vg2j_14mr_1979191t131727::1.0        ./data/vg2j_14mr_1979191t131727.xml 
```
and the corresponding data and browse collections do contain these files. 

However, the collection_data.csv lists: 
`P,urn:nasa:pds:voyager2_rss_jupiter_raw:data:vg2j_14mr_1979191t131728::1.0`
ending in 8, rather than 7.

(The collection_browse.csv does list the correct file:
`P,urn:nasa:pds:voyager2_rss_jupiter_raw:browse:vg2j_14mr_1979191t131727::1.0`)

### 🕵️ Expected behavior

I expected Validate to at least flag a warning, but everything passed. Validate ought to verify the completeness and accuracy of collection inventories.

Request that Validate flags a warning about referential integrity if the collection CSV members and files in the collection do not match.

### 📜 To Reproduce

Bundle can be found at https://pds-rings.seti.org/review-data/voyager_rss_jupiter_raw/

### 📚 Version of Software Used

 Validate 3.0.3, Validate 3.2.0

**Comment by jordanpadams** [comment]
@mace-space would you mind attaching the run log if you still have it?

**Comment by mace-space** [comment]
I don't think it is an issue with Validate after all, but rather due to the options chosen? I attach two logs:
- [validation_rpt_vg2.txt](https://github.com/NASA-PDS/validate/files/11637753/validation_rpt_vg2.txt), which doesn't flag any warnings and uses `-t` and doesn't appear to actually run any referential integrity checks
- [vg2-rss-jupiter-validate-02052023.log](https://github.com/NASA-PDS/validate/files/11637760/vg2-rss-jupiter-validate-02052023.log), which I've just run using ` -R pds4.bundle -v 2` and does flag warnings

**Comment by jordanpadams** [comment]
@mace-space correct. that is the issue. for production validation, you pretty much want to always use the `pds4.bundle` rule, unless you are running validate multiple times in a pipeline of some kind. let us know if you run into any other issues.
